src/apis/article/getArticleByUserId.ts

```diff
@@ -0,0 +1,20 @@
+import { axiosInstance } from "@apis/axiosInstance";
+
+import { GetArticlesResponse } from "@type/apis/articles/GetArticles";
+
+import { DOMAIN } from "@constants/api";
+
+export const getArticleByUserId = async (
+  userId: string,
+  offset: number,
+  limit: number
+) => {
+  const { data } = await axiosInstance.get<GetArticlesResponse>(
+    DOMAIN.GET_ARTICLE.BY_USER_ID(userId),
+    {
+      params: { offset, limit }
+    }
+  );
+
+  return data;
+};
```

Comment:
여기도 api요청 시 토큰이 필요하지 않네요
```suggestion
export const getArticleByUserId = async (
  userId: string,
  offset: number,
  limit: number
) => {
  const { data } = await axiosInstance.get<GetArticlesResponse>(
    DOMAIN.GET_ARTICLE.BY_USER_ID(userId),
    {
      params: { offset, limit },
      useAuth: false
    }
  );

  return data;
};
```